language/German/strings.po

```diff
@@ -1749,6 +1782,7 @@ msgstr "Speichere Status"
 #: src/frontend/mame/ui/state.cpp:539
 msgid "Press a key or joystick button, or select state to overwrite"
 msgstr ""
+"Taste oder Joystick-Knopf drücken, oder Status zum Überschreiben auswählen"
```

Comment:
There is something wrong with the formatting here... it should be

`msgstr "....."`

Reply:
Should be fixed

Reply:
There’s nothing wrong with the formatting there.  If the message is long, the tools will put an empty string component on the line following the keyword.  The components are all concatenated anyway.